index.html

```diff
@@ -14,10 +14,12 @@
 </head>
 <body> 
   <div class="homepage">
-      <header>
```

Comment:
Looks good! 👍  I was just thinking now that maybe we don't need this div with a class of homepage? Maybe the nav element could be inside the header element and give the header  element the class of 'homepage' instead? Not sure if this will clash with anything in the css but worth checking if now that div is not necessary, as it will be more semantic 🤓